.github/workflows/swift-toolchain.yml

```diff
@@ -1732,75 +1642,67 @@ jobs:
       - name: Configure LLVM
         if: matrix.os != 'Android' || inputs.build_android
         run: |
-          # NOTE: used by `matrix.cc`
-          $CLANG_CL = "cl"
+          if ("${{ inputs.build_os }}" -eq "Windows") {
+            $HostToolchain = cygpath -m "${{ github.workspace }}/BinaryCache/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain"
+            $InstallDir = cygpath -m "${{ github.workspace }}/BuildRoot/Library/Developer/${{ matrix.sdk_install_dir }}"
+          } else {
+            $HostToolchain = "${{ github.workspace }}/BinaryCache/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain"
+            $InstallDir = "${{ github.workspace }}/BuildRoot/Library/Developer/${{ matrix.sdk_install_dir }}"
+          }
           if ("${{ matrix.os }}" -eq "Android") {
-            $NDKPATH = cygpath -m ${{ steps.setup-ndk.outputs.ndk-path }}
-            # Since win/arm64 doesn't have one, this logic is necessary because
-            # passing an empty CMAKE_ANDROID_NDK value causes a failure.
-            $CMAKE_NDK_FLAG = "-DCMAKE_ANDROID_NDK=$NDKPATH"
-            $SWIFT_NDK_FLAG = "-DSWIFT_ANDROID_NDK_PATH=$NDKPATH"
+            $NDKPATH = cygpath -m "${{ steps.setup-ndk.outputs.ndk-path }}"
           }
+
           cmake -B ${{ github.workspace }}/BinaryCache/llvm `
                 -D CMAKE_BUILD_TYPE=Release `
                 -D CMAKE_C_COMPILER=${{ matrix.cc }} `
                 -D CMAKE_C_FLAGS="${{ matrix.cflags }}" `
                 -D CMAKE_CXX_COMPILER=${{ matrix.cxx }} `
                 -D CMAKE_CXX_FLAGS="${{ matrix.cxxflags }}" `
-                -D CMAKE_INSTALL_PREFIX=${{ github.workspace }}/BuildRoot/Library/Developer/Platforms/${{ matrix.os }}.platform/Developer/SDKs/${{ matrix.os }}.sdk/usr `
+                -D CMAKE_INSTALL_PREFIX=${InstallDir} `
```

Comment:
Why does this need to be `cygpath`'ed suddenly?

Reply:
A bad update on my part. Anyway, this is now entirely set up from the matrix.